### PR TITLE
Reorganized Community page, added calendar, clarified Slack

### PR DESCRIPTION
**Pull request by oaktowner**
Clarified

### CODE REVIEW

File: content/about/community/join/index.md [hunk 1]
```diff
@@ -10,23 +10,30 @@ to join us and get involved!
 There are quite a few ways to get plugged in:
 
 {{< community_item logo="./discourse.svg" alt="Discourse" >}}
-Head for the [Istio discussion board](https://discuss.istio.io) to participate in discussions
-and get help troubleshooting problems.
+Head for the [Istio discussion board](https://discuss.istio.io) to ask questions, 
+participate in discussions, and get help troubleshooting problems. You can use your github ID to login!
```
Comment (geeknoid) [inline content/about/community/join/index.md]: I don't know if you care, but users can also use a Google account, Twitter, or Yahoo.
'github' -> GitHub.

File: content/about/community/join/index.md [hunk 2]
```diff
@@ -10,23 +10,30 @@ to join us and get involved!
...
+{{< community_item logo="./gcal.svg" alt="Community Calendar" >}}
+Attend events on our [Community calendar](https://calendar.google.com/calendar/embed?src=i10ogf58krfbrsjai5qi16g4do%40group.calendar.google.com&ctz=America%2FLos_Angeles).
 {{< /community_item >}}
 
 {{< community_item logo="./slack.svg" alt="Slack" >}}
-Hangout on [Istio's Slack](https://slack.istio.com) to interact with the community in real-time.
+If you're contributing code to Istio (or thinking about contributing!), you can join us on the 
+[Istio's Slack](https://slack.istio.com).
+Fill out [this form](https://docs.google.com/forms/d/e/1FAIpQLSfdsupDfOWBtNVvVvXED6ULxtR4UIsYGCH_cQcRr0VcG1ZqQQ/viewform)
+to to join.
```
Comment (geeknoid) [inline content/about/community/join/index.md]: to to -> to

File: content/about/community/join/index.md [hunk 3]
```diff
@@ -10,23 +10,30 @@ to join us and get involved!
 There are quite a few ways to get plugged in:
 
 {{< community_item logo="./discourse.svg" alt="Discourse" >}}
-Head for the [Istio discussion board](https://discuss.istio.io) to participate in discussions
-and get help troubleshooting problems.
+Head for the [Istio discussion board](https://discuss.istio.io) to ask questions,
```
Comment (rcaballeromx) [inline content/about/community/join/index.md]: Trailing whitespace
```suggestion
Head for the [Istio discussion board](https://discuss.istio.io) to ask questions,
```

File: content/about/community/join/index.md [hunk 4]
```diff
@@ -10,23 +10,30 @@ to join us and get involved!
...
+on deploying, configuring, and using Istio.
+{{< /community_item >}}
+
+{{< community_item logo="./gcal.svg" alt="Community Calendar" >}}
+Attend events on our [Community calendar](https://calendar.google.com/calendar/embed?src=i10ogf58krfbrsjai5qi16g4do%40group.calendar.google.com&ctz=America%2FLos_Angeles).
 {{< /community_item >}}
 
 {{< community_item logo="./slack.svg" alt="Slack" >}}
-Hangout on [Istio's Slack](https://slack.istio.com) to interact with the community in real-time.
+If you're contributing code to Istio (or thinking about contributing!), you can join us on the
```
Comment (rcaballeromx) [inline content/about/community/join/index.md]:
```suggestion
If you're contributing code to Istio (or thinking about contributing!), you can join us on the
```

File: content/about/community/join/index.md (hunk 1 above)
Comment (rcaballeromx) [inline content/about/community/join/index.md]:
```suggestion
participate in discussions, and get help troubleshooting problems. You can use your GitHub ID to login!
```

Comment (oaktowner) [reply content/about/community/join/index.md]: Yes will do.

File: content/about/community/join/index.md (hunk 2 above)
Comment (oaktowner) [reply content/about/community/join/index.md]: good catch!

**Comment by geeknoid** [comment]
Two more anal-retentive errors to address:

./content/about/community/join/index.md:13: MD009 Trailing spaces
./content/about/community/join/index.md:35: MD009 Trailing spaces

**Pull request merged**